src/Sylius/Behat/Page/Shop/Cart/SummaryPageInterface.php

```diff
@@ -68,6 +68,8 @@ public function hasItemWithOptionValue(string $productName, string $optionName,
 
     public function hasProductOutOfStockValidationMessage(ProductInterface $product): bool;
 
+    public function hasProductOutOfStockFlashMessage(): bool;
```

Comment:
"Flash message" is an implementation detail, that is relevant to the value of the feature. I believe it can be just `hasProductOutOfStockMessage` or even `isProductOutOfStock`, as it's done with `isEmpty` below (and it's obviously handled as flash message in the implementation)